css/design_system/components/footer.scss

```diff
@@ -0,0 +1,109 @@
+ /**
+ * @title Simple Footer
+ * @category components
+ * @component_type footer
+ * @value s-footer
+ * @description Simple footer with paragraph text only
+ *
+ * @example
+ * <footer class="s-footer">
+ *  <div class="s-footer-content">
+ *    <p>Turing School of Software and Design, a Colorado Non-Profit Organization.</p>
+ *  </div>
+ * </footer>
+ */
+
+.s-footer {
+  @extend .s-p-4;
+  @extend .s-bg-gray-800;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.s-footer-content {
+  p {
+    @extend .s-m-0;
+    @extend .s-text-gray-100;
```

Comment:
When using in turing.edu: It's no longer working with the paragraph in the `s-footer-content`, because more specificity on the regular paragraphs. Would this be an appropriate place to use `!important` and re-write the declaration rather than using the `@extend`?

Reply:
We decided to not worry about this on the Savile side; instead adjust where the `s-content` class lives in Turing.edu to avoid this conflict.